cmd/cloudshell_open/appfile.go

```diff
@@ -113,19 +116,35 @@ func promptEnv(list map[string]env) ([]string, error) {
 	// field and prompt the questions as they appear in the app.json file as
 	// opposed to random order we do here.
 	for k, e := range list {
-		var resp string
-		if err := survey.AskOne(&survey.Input{
-			Message: fmt.Sprintf("Value of %s environment variable (%s)",
-				color.CyanString(k),
-				color.HiBlackString(e.Description)),
-			Default: e.Value,
-		}, &resp,
-			survey.WithValidator(survey.Required),
-			surveyIconOpts,
-		); err != nil {
-			return nil, fmt.Errorf("failed to get a response for environment variable %s", k)
+		_, isPresent := existing[k]
```

Comment:
I think `promptEnv` (by its name) "check for existing stuff and generate "secret" env vars" is the wrong place to do it

Let's create a new `prepEnv` method that does the changes you added below, and calls into `promptEnv` actually for prompting purposes only.